moveit_planners/pilz_industrial_motion_planner/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.1.3)
 project(pilz_industrial_motion_planner)
 
 ## Add support for C++11, supported in ROS Kinetic and newer
-add_definitions(-std=c++11)
+add_definitions(-std=c++14)
```

Comment:
This is unrelated and should not be in the same PR/commit. Otherwise it's a good change though.

Reply:
This is necessary to use std::make_unique as that function was added later (c++ 14) to the standard than std::make_shared (c++ 11) for some reason. Do you agree that this can be changed here?

Reply:
I see. Just leave it in then. But I would appreciate if you could update the comment above the line too. ;)

In general it's a good change because our whole code base requires C++14 and the pilz planner was added later but not adapted.